### PR TITLE
feat: added my cart.

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**

### CODE REVIEW

**Comment by vaibhav-mahalle** [comment]
live preview: https://deploy-preview-4--la-gourmet.netlify.app/pages/cart.html

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+            </button>
+            <button class="btn btn-outline-danger">Remove from cart</button>
+          </div>
+        </div>
+
+        <div class="flex-center checkout-details">
+          <div class="p-1 card card-shadow">
+            <div class="font-sm font-bold p-1">Price details</div>
+            <hr>
+            <div class="p-l-1 p-b-1">
```
Comment (vanshitalavange) [inline pages/cart.html:144]: Just a suggestion, try to make these names more understandable

File: home.css [hunk 2]
```diff
@@ -35,7 +35,7 @@
 
 .grid-4-col{
     display: grid;
-    grid-template-columns:repeat(auto-fill, minmax(20rem, 1fr));
+    grid-template-columns:repeat(auto-fit, minmax(20rem, 1fr));
     place-content: center ;
     column-gap: 1rem;
     row-gap: 1rem;
```
Comment (vanshitalavange) [inline home.css:41]: you can use the shorthand property for row gap and column gap

**Review by vanshitalavange — COMMENTED** [review]
Everything looks fine, just look into the comments

File: pages/cart.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link
+      rel="stylesheet"
+      href="https://dexterous-ui.netlify.app/stylesheets/main.css"
+    />
+    <link rel="stylesheet" href="/stylesheets/cart.css" />
+    <link rel="stylesheet" href="/stylesheets/authentication.css" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (shuntzhere) [inline pages/cart.html:11]: You can add these stylesheets to main.css

File: pages/cart.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      <div class="flex-row">
+        <div class="cart-input flex-row">
+          <input type="text" placeholder="search" />
+          <a class="btn-link-noBorder" href="#">
+            <span class="material-icons">search</span>
+          </a>
+        </div>
+        <div class="button-box flex-row">
+          <div class="box-pill">
+            <a class="btn btn-link-noBorder" href="">
```
Comment (shuntzhere) [inline pages/cart.html]: Do not leave href blank. Use #

**Review by shuntzhere — COMMENTED** [review]
The utility classes look good, so does the structure. Just suggested some minor changes. Do give them a look.

File: pages/cart.html (hunk 4 above)
Comment (vaibhav-mahalle) [reply pages/cart.html]: fixed it in the latest commit.

File: home.css (hunk 2 above)
Comment (vaibhav-mahalle) [reply home.css:41]: will keep that in mind.

File: pages/cart.html (hunk 1 above)
Comment (vaibhav-mahalle) [reply pages/cart.html:144]: will make them more readable in refactor.

**Pull request merged**